src/main/java/seedu/duke/manager/Manager.java

```diff
@@ -31,21 +31,16 @@ public abstract class Manager {
     // For this Manager's feature, what commands exist?
     protected ArrayList<Command> supportedCommands;
 
-    // For this Manager's feature, what features (event driver) does it support?
```

Comment:
Yes it can be removed at this moment as it is not very relevant for individual features.